res/layout/edit_event.xml

```diff
@@ -70,19 +78,33 @@
 
                 <!-- BOTTOM HALF GUESTS/ATTENDEES -->
                 <include
-                    layout="@layout/edit_event_2"
                     android:id="@+id/attendees_group"
+                    layout="@layout/edit_event_2"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content" />
             </LinearLayout>
 
-            <View android:id="@+id/color_chip"
+            <View
+                android:id="@+id/color_chip"
                 android:layout_width="25dip"
                 android:layout_height="25dip"
-                android:visibility="gone"
+                android:layout_gravity="start|top"
                 android:layout_marginStart="0dip"
                 android:layout_marginTop="0dip"
-                android:layout_gravity="start|top"/>
+                android:visibility="gone" />
         </FrameLayout>
     </ScrollView>
-</LinearLayout>
+
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
why this?

Reply:
See here:
https://github.com/Etar-Group/Etar-Calendar/pull/736#issuecomment-659226086